bi_sql_editor/models/bi_sql_view.py

```diff
@@ -122,12 +122,13 @@ class BiSQLView(models.Model):
 
     @api.model
     def _default_parent_menu_id(self):
-        return self.env.ref("bi_sql_editor.menu_bi_sql_editor")
+        return self.env.ref(
+            "bi_sql_editor.menu_bi_sql_editor", raise_if_not_found=False
+        )
 
     parent_menu_id = fields.Many2one(
         string="Parent Odoo Menu",
         comodel_name="ir.ui.menu",
-        required=True,
```

Comment:
Hi. please, don't remove this one, as it is mandatory.

otherwise, user can create / update sql views without parent menu field, that doesn't makes sense.